### DISCUSSION TITLE
Documentation for Newbies

### DISCUSSION THREAD

**Discussion by unintended-consequences** [Tea-n-Tech/chia-tea#76] [Q&A]
I don't know what I'm doing, but I had to look up and perform these additional steps (on a relatively clean Ubuntu 20.04):

1. Install pip. 
`git clone https://github.com/Chia-Network/chia-blockchain.git -b latest`

2. Update PATH 
`sudo nano /etc/environment` 
and append 
`:/home/USERNAME/.local/bin`

3. Source the file
'source /etc/environment'

I also had these errors:
`ERROR: launchpadlib 1.10.13 requires testresources, which is not installed.
ERROR: chia-blockchain 1.3.0 has requirement click==7.1.2, but you'll have click 8.0.4 which is incompatible.
ERROR: chia-blockchain 1.3.0 has requirement colorama==0.4.4, but you'll have colorama 0.4.3 which is incompatible.
ERROR: chia-blockchain 1.3.0 has requirement concurrent-log-handler==0.9.19, but you'll have concurrent-log-handler 0.9.20 which is incompatible.
ERROR: chia-blockchain 1.3.0 has requirement multidict==5.1.0, but you'll have multidict 6.0.2 which is incompatible.
ERROR: chia-blockchain 1.3.0 has requirement typing-extensions==4.0.1, but you'll have typing-extensions 4.1.1 which is incompatible.`

**Comment by codie3611** [comment]
Hey @unintended-consequences (nice name btw) appreciate that you take the effort to share your experience.

What python did you use to install it? The system python? Is it the same python environment in which you installed chia itself (I usually keep these two separate)?

The errors are the result of the new chia version which collides with some versions of our dependencies. We had this issue before since chia is very restrictive regarding the versions they use, forcing us to fix some things too which in the end collides if they update their dependencies again. It is very easy, will do it later right away. We possibly could add an automatic workflow to the repo taking care of this once a day.

**Comment by unintended-consequences** [comment]
Hello. Thank you!
I installed via python3 - deactivated from the chia VENV first. So I guess the system python. I just followed the FAQ install section. I amended the instructions and used:

`python3 -m pip install chia-tea`

**Reply by codie3611** [reply]
Okay thanks for the info. Will create an issue to add your info to the docs.

**Reply by codie3611** [reply]
I created #77 and #78 based on this discussion
